resources/sass/pages/_cart.scss

```diff
@@ -0,0 +1,212 @@
+.flex {
```

Comment:
Why are you defining  these flex classes for each and every module, instead of mix-in create a common module in the first place